frontend-react/src/styles/_uswds_settings_content.scss

```diff
@@ -0,0 +1,54 @@
+// USWDS Settings that are different from the defaults.
+// see: https://designsystem.digital.gov/documentation/settings/
+
+$rs-width: "desktop-lg";
+
+@use "uswds-core" with (
+    /*
+    * These are necessary for the image and font urls referenced in the source
+    * files to resolve correctly.
+    * these paths are from where the output css lives, in this case
+    * /frontend-react/src/content/generated/global.out.css
+    */
+        $theme-image-path: "@uswds/uswds/img",
```

Comment:
Strange indentation.

Reply:
Unfortunately I don't know why prettier is doing this, but with the project settings we have it automatically adds it.